stan/math/rev/functor/reduce_sum.hpp

```diff
@@ -220,8 +243,12 @@ struct reduce_sum_impl<ReduceFunction, require_var_t<ReturnType>, ReturnType,
       partials[i] = 0.0;
     }
 
+    local_args_tuple_t local_args([&] { return scoped_args_tuple(args...); });
+    // @Steve: This should work, but fails to compile???
```

Comment:
Could you have a look here, @SteveBronder ? I don't quite get why i can't use the argument forward thing from the TBB which fails to compile for me.